server/index.js

```diff
@@ -198,6 +214,32 @@ module.exports = (_config = {}) => {
         .listen(port, () => {
           logger.info('bpanel app running on port', port);
         });
+
+      // can serve over https
+      if (config.bool('bpanel-https', false)) {
```

Comment:
Do you want to document these extra configs by adding them to the sample conf files? 

Also, I'm wondering if the `bpanel-` prefix is necessary since bcfg is already presuming that prefix for the config since that is the name of the module. i.e. I think that if you wanted to pass this in via env variable, you'll end up having to do `BPANEL_BPANEL_HTTPS`

Reply:
`https: true` ?

Reply:
I believe so yes. If it's recognized as in the `bpanel` module, then that should work.